Gemfile

```diff
@@ -26,6 +26,7 @@ gem 'turbolinks'
 
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 1.2'
+gem 'byebug'
```

Comment:
can you add this to the test & development group. We don't want it loading in production :)

Also, I recommend using `pry-byebug`. Similar... but more powerful.

```
  gem 'pry-rails'
  gem 'pry-byebug'
```